2.5/Dockerfile

```
@@ -20,11 +20,13 @@ building and running various Ruby $RUBY_VERSION applications and frameworks. \
 Ruby is the interpreted scripting language for quick and easy object-oriented programming. \
 It has many features to process text files and to do system management tasks (as in Perl). \
 It is simple, straight-forward, and extensible." \
-    PATH=$PATH:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/bin:/opt/rh/$NODEJS_SCL/root/usr/bin \
-    LD_LIBRARY_PATH=/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/lib64:/opt/rh/$NODEJS_SCL/root/usr/lib64 \
-    X_SCLS=rh-ruby$RUBY_SCL_NAME_VERSION \
-    MANPATH=/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/share/man:/opt/rh/$NODEJS_SCL/root/usr/share/man \
-    PYTHONPATH=/opt/rh/$NODEJS_SCL/root/usr/lib/python2.7/site-packages
+    PATH=$PATH:/opt/rh/$NODEJS_SCL/root/usr/bin:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/local/bin:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/bin:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/bin:/opt/rh/$NODEJS_SCL/root/usr/bin \
```

One additional question: Why is it not quoted? I'm not entirely sure this works reliable with spaces/special characters.

This would be more resilient:
```suggestion
    PATH="$PATH:/opt/rh/$NODEJS_SCL/root/usr/bin:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/local/bin:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/bin:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/bin:/opt/rh/$NODEJS_SCL/root/usr/bin" \
```

---

It isn't necessary, whitespaces work like delimiters. I don't see the reason why it shouldn't work. Or I am mistaken?

---

In Dockerfile, we do not need double quotes. I did not see this usage in Dockerfile. From my point of view, it does not make sense.

---

Well, `Dockerfile` actually passes it to shell (env), it's not a docker specific thing. But you're right it's most probably not needed (at least when there're no spaces/special chars), so feel free to ignore this.

It was just to point out - something to be considered, maybe for next time. Using quotes is simply more resilient. IOW you're right you don't need it, until someone uses `!` f.e. and then it breaks. Shouldn't be our case though.